swiftgen-cli/base.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  base.swift
+//  SwiftGen
+//
+//  Created by Peter Livesey on 11/10/16.
+//  Copyright © 2016 AliSoftware. All rights reserved.
+//
+
+import Commander
+import PathKit
+import GenumKit
+import Stencil
+
+let baseCommand = command(
```

Comment:
Why the need for that command?

`swiftgen templates cat json-base` already covers outputting the verbatim content of a template…